### PR TITLE
OPA filters: Reduce default buffer size for reading the requests' body

### PR DESCRIPTION
**Pull request by mjungsbluth**
This PR

* reduces the default buffer size for reading the request body to 8KB
* and makes it configurable via command line and Skipper config

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -513,6 +514,7 @@ func NewConfig() *Config {
 	flag.DurationVar(&cfg.OpenPolicyAgentCleanerInterval, "open-policy-agent-cleaner-interval", openpolicyagent.DefaultCleanIdlePeriod, "Duration in seconds to wait before cleaning up unused opa instances")
 	flag.DurationVar(&cfg.OpenPolicyAgentStartupTimeout, "open-policy-agent-startup-timeout", openpolicyagent.DefaultOpaStartupTimeout, "Maximum duration in seconds to wait for the open policy agent to start up")
 	flag.Int64Var(&cfg.OpenPolicyAgentMaxRequestBodySize, "open-policy-agent-max-request-body-size", openpolicyagent.DefaultMaxRequestBodySize, "Maximum number of bytes from a http request body that are passed as input to the policy")
+	flag.Int64Var(&cfg.OpenPolicyAgentBodyReadBufferSize, "open-policy-agent-body-read-buffer-size", openpolicyagent.DefaultBodyBufferSize, "Read buffer size for the request body")
```
Comment (AlexanderYastrebov) [inline config/config.go:517]: Maybe we can make flag, option and constant names consistent,
e.g. `open-policy-agent-body-buffer-size` or `DefaultBodyReadBufferSize`.
Also maybe it makes sense to highlight that it is request body, e.g. similar to OpenPolicyAgentMaxRequestBodySize make it `OpenPolicyAgentRequestBodyBufferSize` (and `open-policy-agent-request-body-buffer-size`)

Comment (AlexanderYastrebov) [inline config/config.go:517]: Config test needs to be adjusted due to non-zero default:
```
--- FAIL: Test_NewConfigWithArgs (0.01s)
    --- FAIL: Test_NewConfigWithArgs/test_only_valid_flag_overwrite_yaml_file (0.01s)
        config_test.go:361: config.NewConfig() want vs got:
              &config.Config{
              	... // 1 ignored and 224 identical fields
              	OpenPolicyAgentCleanerInterval:      s"10s",
              	OpenPolicyAgentStartupTimeout:       s"30s",
            - 	OpenPolicyAgentBodyReadBufferSize:   8192,
            + 	OpenPolicyAgentBodyReadBufferSize:   0,
              	OpenPolicyAgentMaxRequestBodySize:   1048576,
              	OpenPolicyAgentMaxMemoryBodyParsing: 104857600,
              	PassiveHealthCheck:                  {},
              }
```

Comment (mjungsbluth) [reply config/config.go:517]: Good points, will adjust...

**Comment by szuecs** [comment]
:+1:

**Pull request merged**